fastly/block_fastly_service_logging_gcs.go

```diff
@@ -151,6 +156,7 @@ func (h *GCSLoggingServiceAttributeHandler) Create(_ context.Context, d *schema.
 		Name:             gofastly.String(resource["name"].(string)),
 		Path:             gofastly.String(resource["path"].(string)),
 		Period:           gofastly.Int(resource["period"].(int)),
+		ProjectID:        gofastly.String(resource["project-id"].(string)),
```

Comment:
I think this is a typo...

```suggestion
		ProjectID:        gofastly.String(resource["project_id"].(string)),
```

Reply:
Thank you for catching that, fixed in 3e942ee27f88d4abcf4a5b16d924ff58bf0aec84.